hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/ingest/IngestChecker.java

```diff
@@ -184,7 +186,7 @@ public TransactionInfo runAllChecks(
         }
 
         // 6. Verify payer's signatures
-        verifyPayerSignature(txInfo, payerKey);
+        verifyPayerSignature(txInfo, payer, payerKey, configuration);
```

Comment:
nit: Do we need both payer and payerKey as inputs or just payer should be fine?

Reply:
Good catch! Fixed.